service/server.py

```diff
@@ -67,21 +68,38 @@ def get_properties_for_address(address):
     return query.execute().hits
 
 
+def paginated_address_records(address_records, page_number):
+    start_index = (page_number-1)*int(SEARCH_RESULTS_PER_PAGE)
```

Comment:
let's raise a chore to look at env variables in general: i followed how a couple of others handled config on that point, but agreed that this is not good